eth-contracts/contracts/DelegateManager.sol

```diff
@@ -1045,6 +1077,16 @@ contract DelegateManager is InitializableV2 {
         });
     }
 
+    /**
+     * @notice Update total amount delegated from an address
+     * @param _delegator - address of service provider
+     * @param _amount - updated lock up amount
+     */
+    function _updateDelegatorTotal(address _delegator, uint256 _amount) internal
```

Comment:
so this is called in conjunction with `_updateDelegatorStake()` every single time except for in `slash()` and `_distributeDelegateRewards()`. does it make sense to just call it inside there instead of separately everywhere? would be less error prone + logically makes sense that this is a step in "updating delegator stake"
wdyt?

Reply:
yeah this is an interesting point...the reason it's not used in slash and distribute is an optimization where we don't need to update them together _but_ we do need to update each delegator total in slash/distribute so separating this kind of made more sense

Reply:
per offline discussion - modified `_updateDelegatorStake` to call into `_updateDelegatorTotalStake` helper, while distribute/slash are still able to call `_updateDelegatorTotalStake` directly